### PR TITLE
some text improvements

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by RobFog** [comment]
* improvements

**Comment by Jojo-Schmitz** [comment]
As long as typos are just in the commit titel and -comment... anyway, fixed

File: mscore/prefsdialog.ui [hunk 1]
```diff
@@ -1109,9 +1109,6 @@
           <item row="0" column="0">
            <widget class="QCheckBox" name="drawAntialiased">
             <property name="toolTip">
-             <string>Disable antialiased drawing</string>
```
Comment (anatoly-os) [inline mscore/prefsdialog.ui]: Are you sure? Tooltip is exactly a tooltip in this case which gives a bit more information. The new string explains why a user needs to do so. The latter is important, but not a tooltip by the meaning.

Comment (Jojo-Schmitz) [reply mscore/prefsdialog.ui]: this tooltip is just superfluos, the tick box is labled "Use antialiasing" so I see not valuable extra information in that tooltip. Also it doesn't disable but toggle the use of antialiasing, so the tooltip is even wrong 50% of the time

Comment (Jojo-Schmitz) [reply mscore/prefsdialog.ui]: and the new tooltip is `Uncheck this to speed up drawing`, much more useful. We're not using `whatsThis` anywhere else (and I don't even know what they are good for)

**Pull request merged**